x-pack/solutions/security/packages/kbn-securitysolution-io-ts-list-types/src/common/meta/index.ts

```diff
@@ -7,7 +7,7 @@
 
 import * as t from 'io-ts';
 
-export const meta = t.object;
+export const meta = t.UnknownRecord;
```

Comment:
This is awesome. Thanks for digging into it and ensuring the type is properly defined.